Clients/GUI/Shared/NewsComponent.razor

```diff
@@ -0,0 +1,109 @@
+﻿@page  "/NewsComponent"
+
+    <div class="container">
+        @if (errorMessage == "")
+        {
+            <h1>@newsEconomics[0].Category</h1>
+            <div class="row row-margin-bottom">
+                @for (int i = 0; i < 2; ++i)
```

Comment:
Why 2?

Reply:
Because Spartak said, that he wants 2 news in row in same category

Reply:
> Because Spartak said, that he wants 2 news in row in same category

Ok then